plugins/micromega/coq_micromega.ml

```diff
@@ -1875,6 +1882,8 @@ let micromega_tauto ?(abstract=true) pre_process cnf spec prover
     let res' = compact_proofs prover spec.coeff_eq cnf_ff res cnf_ff' in
     let ff', res', ids = (ff', res', Mc.ids_of_formula Mc.IsProp ff') in
     let res' = dump_list spec.proof_typ spec.dump_proof res' in
+    if show_used_hyps ()
+    then Feedback.msg_info Pp.(str "Used hypotheses:"++pr_enum Names.Id.print ids);
```

Comment:
maybe worth being a bit more explicit about what used the hypotheses, ie
```suggestion
    then Feedback.msg_info Pp.(str "Micromega used hypotheses:"++pr_enum Names.Id.print ids);
```
also maybe worth having a space after `:`